java/gradle.java/test/unit/src/org/netbeans/modules/java/j2seplatform/api/Bundle_da.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
da = yes in slavic languages, it's a minor, but can't we name it Bundle_yes then? At first I thought that this is for Danish locale.